x-pack/legacy/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -320,12 +330,13 @@ export function App({
                     persistedDoc: newDoc,
                     lastKnownDoc: newDoc,
                   }));
-
                   if (docId !== id) {
                     redirectTo(id);
                   }
                 })
-                .catch(() => {
+                .catch(e => {
+                  // eslint-disable-next-line no-console
+                  console.dir(e);
```

Comment:
Not really: right now every exception is caught and a generic error "Unable to save document" is displayed, even if the error originates from parsing dashboard URL. I don't want to change this behavior now without figuring out proper error messages to be displayed to the user (if any), but thought it would be useful to have the error at least logged in case it occurs.